.ci/bump-elastic-stack-snapshot.yml

```diff
@@ -20,13 +20,12 @@ scms:
   default:
     kind: github
     spec:
-      user: '{{ requiredEnv "GIT_USER" }}'
```

Comment:
I tested a similar configuration:
- https://github.com/v1v/updatecli-demo/commit/3865c6e6399f8ada8cfb9ed9dd075a45b8fe65a2

and produced the expected [outcome](https://github.com/v1v/updatecli-demo/pull/42/commits)

<img width="852" alt="image" src="https://github.com/elastic/apm-server/assets/2871786/3bde67cd-2b60-4357-9224-4d7a4c9ccb01">